client/app/assets/less/inc/schema-browser.less

```diff
@@ -1,102 +1,107 @@
-div.table-name {
```

Comment:
This change feels bigger than the others. Can you summarize what the expected changes are to the schema browser?

Reply:
Changes expected are basically none. Just refactored the styles to make it easier to work with, as many of the positionings were done in a workaround-like manner. There were bugs as well: I've changed the way tooltips are positioned upon these components to make it a11y (and UI overall) friendly.

Reply:
Quick summary of the changes:

- Improved nesting
- Decoupled the button to open the drawer with the one that adds it to the editor
- Updated the box sizing to flexbox to achieve the same result more consistently
- Used flexbox to center, in place of paddings and margins
- Reduced repetition re-structuring how classes are set
- Added focus feedback
- Added transition
- TODO (not tracked): render of drawer has no transition